meta-lmp-base/classes/fip-utils.bbclass

```diff
@@ -153,7 +106,9 @@ python () {
                     d.appendVar('FIP_DEVICETREE', items[1] + ',')
                     break
     if d.getVar('FIP_SIGN_ENABLE') == '1':
-        generate_sign_key_path(d)
+        signature_key = d.getVar('FIP_SIGN_KEY')
```

Comment:
It is a PATH, this was to be similar to OPTEE_TA_SIGN_KEY. But we can have UBOOT_SIGN_KEYNAME + UBOOT_SIGN_KEYDIR, would that be preferred?

Reply:
> But we can have like UBOOT_SIGN_KEYNAME + UBOOT_SIGN_KEYDIR, would that be preferred?
works for me, thanks!

Reply:
Since all tooling expects to use just the key (and not like mkimage, which uses the folder as input), I will just change to KEY_PATH as you suggested.

Reply:
Done.